test/integration/source-default/source_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+describe directory '/etc/haproxy' do
+  it { should exist }
+end
+
+describe file '/etc/haproxy/haproxy.cfg' do
+  its(:mode) { should cmp '0644' }
+end
+
+if os.debian? && os.release.start_with?('14')
```

Comment:
why is this different from others? why would it not be enabled on 14?

Reply:
relic of the past - we don't even support 14, dropping this